common/achievement/util.ts

```diff
@@ -11,6 +11,8 @@ import { Achievement_template, User_achievement } from '../../graphql/generated'
 import { renderTemplate } from '../../utils/helpers';
 import { getLogger } from '../logger/logger';
 
+const courseDefaultImage =
```

Comment:
Instead of copying this line, we could use [getCourseImageURL](https://github.com/corona-school/backend/blob/cb7425895d827b4b756cd3e47cfdac4221f0bbfa/common/courses/util.ts#L15-L17C2) directly

Reply:
Oh thank you. This realy is the easier way.

Reply:
Oh thank you. This realy is the easier way.